cob_bringup_sim/scripts/spawn_object_jungheinrich.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
```

Comment:
the file is needed. but we probably should rename it. @ipa-nhg since this is your code, do you have a suggestion how to name it?